Remove user agent for ios (#3083)
segmentio/segment-docs: src/connections/spec/common.md

| screen.height            |              | ✅             | ✅                 |
| screen.width             |              | ✅             | ✅                 |
| traits                   |              | ✅             | ✅                 |
| userAgent                | ✅            |              | ✅                 |
| timezone                 |              | ✅             | ✅                 |

> - IP Address is not collected by Segment's libraries, but instead filled in by Segmen'ts servers when it receives a message for **client side events only**.

Reviewer: ```suggestion
- IP Address isn't collected by Segment's libraries, but instead filled in by Segment's servers when it receives a message for **client side events only**.
```
